src/test/codewhisperer/explorer/codewhispererNode.test.ts

```diff
@@ -10,6 +10,17 @@ import { codewhispererNode } from '../../../codewhisperer/explorer/codewhisperer
 import { AuthUtil } from '../../../codewhisperer/util/authUtil'
 
 describe('codewhispererNode', function () {
+    let isConnectionValid: sinon.SinonStub
+    let isConnected: sinon.SinonStub
+
+    beforeEach(function () {
+        isConnectionValid = sinon.stub(AuthUtil.instance, 'isConnectionValid')
```

Comment:
This block of code sets lots of auth stuff to not being connected initially, any following test that needs it to be connected (returns true) will set it on its own.